Formula/shadowsocks-rust.rb

```diff
@@ -20,6 +20,33 @@ class ShadowsocksRust < Formula
 
   def install
     system "cargo", "install", *std_cargo_args
+    (buildpath/"shadowsocks-rust.json").write <<~EOS
```

Comment:
If upstream doesn't ship config neither should we.

Reply:
Shadowsocks-rust has [example config](https://github.com/shadowsocks/shadowsocks-rust/blob/master/examples/config.json), so it should be added.

And shadowsocks-libev [also have config](https://github.com/Homebrew/homebrew-core/blob/821c95a447495edcd72fb1a9eddb6c1275d240c3/Formula/shadowsocks-libev.rb#L43).

Reply:
Then please install that config.